package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@code.gov/code-gov-front-end",
-  "version": "1.2.0",
+  "version": "1.2.1",
```

Comment:
@seanstar12 Wondering why this bumped here. Wouldn't it already be on the front-end from when I did the NPM bump a little while ago? Not that it really matters when this shows up.